google-cloud-dlp-v2/lib/google/cloud/dlp/v2/version.rb

```diff
@@ -21,7 +21,7 @@ module Google
   module Cloud
     module Dlp
       module V2
-        VERSION = "0.7.0"
```

Comment:
Where did this version number come from?  0.0.1 was the version in https://github.com/googleapis/googleapis-gen/blob/master/google/privacy/dlp/v2/google-cloud-privacy-dlp-v2-ruby/lib/google/cloud/dlp/v2/version.rb.

Reply:
The microgenerator always generates version 0.0.1. The release process then updates the version in place when we release new versions. To prevent that version from being overwritten when the library is regenerated, Ruby's merge function for synthtool identifies and does not copy the version file if one is already present.

Reply:
In other words, the version file probably belongs in the "preserve" regex.

Reply:
Sounds good.  Except the current implementation of "preserve" actually preserves the non-existence of files as well, so preserving version.rb means it would never be copied in.  I imagine we'd want to copy it in if it doesn't exist, no?

Reply:
Yes, correct.